build/index.ts

```diff
@@ -568,6 +569,8 @@ export async function buildDocument(
 
   wrapTables($);
 
+  injectBanners($, CONTENT_ROOT, doc.locale, metadata);
```

Comment:
We can assume `CONTENT_ROOT` inside `injectBanners`.

```suggestion
  injectBanners($, doc.locale, metadata);
```